web-bundle/src/main/java/com/graphhopper/http/TruckFlagEncoder.java

```diff
@@ -19,7 +19,7 @@ public static VehicleEncodedValues createTruck() {
         BooleanEncodedValue accessEnc = new SimpleBooleanEncodedValue(
                 EncodingManager.getKey(TRUCK_VEHICLE_NAME, "access"), true);
         DecimalEncodedValue speedEnc = new DecimalEncodedValueImpl(
-                EncodingManager.getKey(TRUCK_VEHICLE_NAME, "average_speed"), TRUCK_SPEED_BITS, TRUCK_SPEED_FACTOR, false);
```

Comment:
@michaz I originally hit an error during `import` that I tracked down to this field (`storeTwoDirections`), and changing it to `true` fixed the error. I wanted to double check with you that it's safe to update here, because I know very little about how these encoded values are expected to work under the hood (having the error go away seemed like a good step, though!)

 The full error was 
```
Caused by: java.lang.IllegalArgumentException: truck_average_speed: value for reverse direction would overwrite forward direction. Enable storeTwoDirections for this EncodedValue or don't use setReverse
	at com.graphhopper.routing.ev.IntEncodedValueImpl.uncheckedSet(IntEncodedValueImpl.java:173)
	at com.graphhopper.routing.ev.DecimalEncodedValueImpl.setDecimal(DecimalEncodedValueImpl.java:105)
	at com.graphhopper.routing.util.parsers.AbstractAverageSpeedParser.setSpeed(AbstractAverageSpeedParser.java:64)
	at com.graphhopper.routing.util.parsers.CarAverageSpeedParser.handleWayTags(CarAverageSpeedParser.java:140)
	at com.graphhopper.routing.util.parsers.AbstractAverageSpeedParser.handleWayTags(AbstractAverageSpeedParser.java:74)
	at com.graphhopper.routing.util.OSMParsers.handleWayTags(OSMParsers.java:101)
	at com.graphhopper.reader.osm.CustomOsmReader.addEdge(CustomOsmReader.java:352)
	at com.graphhopper.reader.osm.CustomWaySegmentParser$Pass2Handler.handleSegment(CustomWaySegmentParser.java:394)
	at com.graphhopper.reader.osm.CustomWaySegmentParser$Pass2Handler.splitSegmentAtSplitNodes(CustomWaySegmentParser.java:364)
	at com.graphhopper.reader.osm.CustomWaySegmentParser$Pass2Handler.splitLoopSegments(CustomWaySegmentParser.java:318)
	at com.graphhopper.reader.osm.CustomWaySegmentParser$Pass2Handler.splitWayAtJunctionsAndEmptySections(CustomWaySegmentParser.java:293)
	at com.graphhopper.reader.osm.CustomWaySegmentParser$Pass2Handler.handleWay(CustomWaySegmentParser.java:267)
	at com.graphhopper.reader.osm.CustomWaySegmentParser$ReaderElementHandler.handleElement(CustomWaySegmentParser.java:524)
	at com.graphhopper.reader.osm.CustomWaySegmentParser.readOSM(CustomWaySegmentParser.java:424)
	```

Reply:
Yes, I found the change in our code that prompted this, and the solution is correct.